pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -19,33 +19,243 @@ package dockershim
 import (
 	"fmt"
 
+	dockertypes "github.com/docker/engine-api/types"
+	dockercontainer "github.com/docker/engine-api/types/container"
+	dockerfilters "github.com/docker/engine-api/types/filters"
+
 	runtimeApi "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
 )
 
+const (
+	defaultSandboxImage = "gcr.io/google_containers/pause-amd64:3.0"
+
+	// Various default sandbox resources requests/limits.
```

Comment:
I think we'll continue using them since they are implementation details of docker.

Good question. I am not sure how we got the default, but I think as long as the sandbox container doesn't consume much resource, the values should be ok.